packages/forms-web-app/src/dynamic-forms/dynamic-components/utils/question-utils.js

```diff
@@ -42,3 +42,10 @@ exports.getListedBuildingForQuestion = (journeyResponse, fieldName) => {
 
 	return listedBuildings.filter((listedBuilding) => listedBuilding.fieldName == fieldName);
 };
+
+/** @type {(maybeDivider: import('../../question-props').Option) => maybeDivider is { divider: string }} */
+exports.optionIsDivider = (maybeDivider) => Object.hasOwn(maybeDivider, 'divider');
```

Comment:
ok this was confusing for me, so divider is a prop we use on options and pass to the govuk macro, and our logic needs to account for there not being a consistent object shape for option

Would it be better if we used a const for the string and reused that in the question instantiation + here?

Reply:
Yeah that's exactly it. Happy to constantify the key, it's only used twice